src/app/core/shell/sidenav/sidenav.component.scss

```diff
@@ -176,3 +176,7 @@
     }
   }
 }
+
```

Comment:
Remove this.

Reply:
replace app.logo at line 135 with :
```suggestion
.app-logo {
      margin: auto;
      &:hover {
        cursor: pointer;
      }
   }
```

Reply:
> replace app.logo at line 135 with :

The new class should be started at line 179? Or just replace this code with the previous one without changing the position?

Reply:
No just add 
&:hover {
        cursor: pointer;
      }
to app.logo at line 135 and remove the new homeIcon id youve created